curl_cffi/requests/session.py

```diff
@@ -31,6 +31,19 @@
 except ImportError:
     pass
 
+if TYPE_CHECKING:
+    from typing_extensions import TypedDict
```

Comment:
typing_extensions in only installed for Python 3.7, will this be a problem?

Reply:
As it's under a `TYPE_CHECKING` block, this code will only run during static type analysis. typing_extensions is a requirement for all type checkers, so this shouldn't be an issue.